blast/formatter/server/internal/mtx/prometheus.go

```diff
@@ -0,0 +1,36 @@
+package mtx
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+var (
+	jobTimes = promauto.NewHistogram(prometheus.HistogramOpts{
+		Name: "format_times",
+		Help: "Formatter job durations in seconds.",
+		Buckets: []float64{
```

Comment:
Wondering why the buckets are not exactly exponential.  I guess for conversion from whole-number seconds to 30/60/120 (minute-based ranges)?  If so, we could still do a little better, e.g. 1, 2, 4, 8, 15, 30, 60?  Or is Bob dictating?

Reply:
That was just because exponential on time ranges didn't make sense to me and I didn't want to overload the grafana UI with a bunch of rows, the mblast one overall is way too crowded and I need to go in and tweak that.  Basically this is just a starting point and as we get data from the service we can tweak from there, like if most lie under 1 second we can zoom in on that and maybe drop some of the higher ones.

![grafana-shot](https://user-images.githubusercontent.com/1297911/162763464-f8d984c2-165f-4668-add2-02961abcfe22.png)
 already